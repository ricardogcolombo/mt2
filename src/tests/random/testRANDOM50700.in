50 700
1 16 18 47 12
1 2 102 3 126
1 15 12 36 130
1 28 112 30 103
1 45 11 6 23
1 12 124 31 24
1 41 71 26 85
1 38 12 33 43
1 27 129 20 120
1 2 22 42 78
1 2 47 26 130
1 29 94 34 88
1 19 22 31 73
1 49 25 44 129
1 25 105 15 110
1 27 23 30 62
1 5 72 21 66
1 34 2 11 102
1 36 109 34 118
1 35 28 37 78
1 32 15 38 38
1 30 10 39 74
1 50 63 41 74
1 42 82 34 28
1 38 62 47 54
1 30 20 16 114
1 2 124 1 83
1 22 33 38 125
1 49 66 15 72
1 36 117 7 122
1 21 4 4 49
1 47 99 9 33
1 44 25 17 109
1 32 40 1 63
1 36 118 49 103
1 43 73 39 68
1 31 112 22 31
1 40 75 16 106
1 47 4 45 114
1 16 38 34 18
1 44 12 22 87
1 39 21 26 89
1 17 46 7 89
1 20 8 14 86
1 42 49 48 16
1 25 119 48 16
1 19 18 33 29
1 1 103 13 25
1 29 72 38 89
1 39 111 50 52
1 42 71 15 78
1 25 58 43 9
1 31 38 7 52
1 3 130 37 96
1 12 70 16 27
1 37 122 9 41
1 12 105 6 94
1 18 80 32 130
1 49 115 12 24
1 39 66 27 116
1 12 76 7 20
1 31 117 50 79
1 27 20 30 96
1 45 24 47 35
1 48 21 47 123
1 14 104 33 38
1 25 36 29 73
1 1 112 48 38
1 16 13 9 45
1 27 117 19 24
1 44 79 6 46
1 5 21 45 46
1 9 54 45 44
1 48 100 16 69
1 15 5 25 100
1 40 38 21 14
1 23 40 40 82
1 29 105 16 102
1 37 81 19 39
1 3 17 41 6
1 16 119 25 7
1 15 89 11 75
1 14 9 15 116
1 35 89 9 55
1 45 42 40 100
1 11 42 7 1
1 49 93 12 47
1 31 130 13 25
1 4 1 50 49
1 29 16 7 128
1 19 114 46 66
1 39 60 40 36
1 27 22 26 56
1 41 106 49 40
1 12 99 24 125
1 8 8 45 32
1 37 76 8 39
1 22 27 39 9
1 38 2 48 83
1 30 115 34 54
1 17 113 47 46
1 2 51 11 41
1 35 30 23 39
1 45 48 22 74
1 24 49 6 39
1 45 72 48 30
1 8 31 17 78
1 1 30 42 33
1 6 58 45 55
1 37 129 21 0
1 16 2 17 118
1 19 101 8 49
1 23 85 48 69
1 36 84 15 12
1 19 6 38 41
1 24 16 8 66
1 27 66 36 2
1 6 47 18 110
1 31 59 18 84
1 9 33 13 21
1 38 116 21 76
1 39 85 34 102
1 25 40 29 99
1 3 31 46 46
1 50 73 28 108
1 33 33 15 72
1 42 17 11 105
1 37 88 39 91
1 37 58 13 90
1 39 66 28 120
1 21 27 5 128
1 29 64 38 1
1 49 82 34 9
1 8 51 9 58
1 38 49 24 54
1 25 15 35 16
1 29 71 11 69
1 40 67 46 62
1 9 34 11 7
1 7 85 18 122
1 33 127 1 74
1 45 6 50 50
1 30 105 19 73
1 31 72 18 23
1 22 82 48 68
1 37 16 38 0
1 48 71 46 37
1 18 21 47 109
1 47 33 4 12
1 44 16 11 52
1 34 21 12 29
1 33 10 38 30
1 22 128 45 114
1 24 29 50 90
1 2 82 3 34
1 48 45 36 110
1 35 74 11 39
1 48 112 29 107
1 46 50 33 9
1 50 67 19 25
1 5 103 35 90
1 11 3 44 80
1 28 104 1 106
1 23 18 38 107
1 6 87 2 32
1 2 89 23 32
1 3 37 49 92
1 34 14 5 77
1 18 19 34 120
1 39 74 8 70
1 23 60 46 96
1 48 75 19 71
1 41 15 39 127
1 33 103 22 127
1 21 123 41 93
1 46 41 29 29
1 40 105 23 44
1 41 59 44 48
1 5 93 46 51
1 46 41 39 59
1 1 67 22 65
1 42 37 1 88
1 22 90 37 110
1 29 14 45 59
1 35 57 1 64
1 24 51 17 50
1 5 91 35 35
1 13 37 46 60
1 17 9 12 104
1 39 55 50 87
1 32 22 11 103
1 5 13 24 44
1 33 121 39 14
1 13 14 50 69
1 45 0 48 107
1 12 31 39 101
1 3 15 26 72
1 21 101 34 40
1 22 23 10 58
1 14 2 41 69
1 46 9 32 81
1 8 75 23 5
1 48 52 15 77
1 38 11 8 112
1 18 122 30 93
1 16 59 30 54
1 41 63 44 108
1 10 5 27 118
1 49 89 42 1
1 5 67 30 20
1 23 41 28 106
1 40 111 35 77
1 16 0 25 52
1 5 17 13 2
1 36 98 8 64
1 30 125 16 70
1 28 79 14 7
1 35 11 9 50
1 28 81 16 45
1 50 123 9 14
1 1 67 3 30
1 15 94 27 96
1 22 112 17 77
1 37 102 46 5
1 50 68 1 77
1 30 93 15 125
1 8 23 3 4
1 30 121 5 95
1 8 59 47 43
1 28 101 12 87
1 36 28 15 112
1 25 13 39 65
1 25 109 45 52
1 30 61 1 109
1 7 56 39 86
1 20 103 9 15
1 22 14 32 12
1 49 118 10 114
1 40 9 38 107
1 1 84 47 111
1 26 116 33 6
1 24 34 1 96
1 16 74 19 52
1 2 7 11 105
1 35 68 48 53
1 16 66 27 118
1 50 18 7 109
1 34 69 20 47
1 17 99 6 5
1 33 120 4 111
1 30 53 3 121
1 9 68 17 84
1 39 92 33 79
1 5 3 42 96
1 2 110 45 116
1 1 12 49 60
1 37 60 40 6
1 26 119 14 24
1 1 119 46 83
1 36 82 48 23
1 35 8 38 51
1 34 76 12 100
1 47 113 44 77
1 45 42 19 115
1 17 6 16 93
1 12 86 24 123
1 10 18 30 118
1 50 89 32 96
1 21 122 49 109
1 40 93 15 19
1 23 11 6 24
1 9 49 49 53
1 6 65 44 76
1 19 82 13 59
1 3 104 35 97
1 42 0 3 21
1 48 119 15 118
1 2 108 44 29
1 29 58 17 86
1 28 8 22 71
1 25 43 13 53
1 45 32 28 12
1 38 61 5 126
1 8 49 29 43
1 2 128 23 126
1 19 13 34 107
1 2 69 45 49
1 16 70 30 128
1 28 87 37 95
1 27 75 9 53
1 5 130 44 120
1 50 83 5 35
1 1 44 46 55
1 31 94 42 57
1 17 16 1 21
1 3 50 46 72
1 13 130 11 22
1 26 72 24 65
1 37 27 8 28
1 27 97 14 122
1 25 130 34 46
1 14 15 21 54
1 15 86 45 14
1 6 79 42 68
1 42 106 21 14
1 27 51 25 82
1 13 68 50 38
1 44 129 9 0
1 42 99 28 17
1 36 79 45 125
1 47 33 36 91
1 21 60 44 98
1 13 129 1 103
1 30 106 3 117
1 36 98 40 122
1 8 54 20 41
1 13 119 3 72
1 2 18 33 11
1 23 102 37 12
1 45 73 33 26
1 34 90 10 7
1 16 11 1 127
1 14 115 36 86
1 35 54 9 58
1 46 8 28 83
1 32 130 12 54
1 34 8 3 20
1 23 70 16 48
1 23 37 45 97
1 13 116 23 45
1 1 98 7 36
1 37 72 25 17
1 44 10 23 66
1 37 112 41 76
1 29 64 12 106
1 9 92 12 106
1 16 117 31 31
1 9 97 1 7
1 35 112 18 47
1 6 74 28 7
1 21 94 1 117
1 18 28 46 35
1 44 84 2 6
1 21 33 30 47
1 49 84 10 70
1 18 29 6 106
1 41 15 15 29
1 7 36 30 46
1 24 6 44 59
1 33 9 19 8
1 17 101 28 26
1 19 84 48 14
1 31 99 37 67
1 25 67 6 41
1 48 126 10 68
1 4 22 42 73
1 35 46 45 1
1 39 101 9 50
1 42 46 17 101
1 45 27 4 80
1 49 39 26 83
1 32 73 33 89
1 36 114 31 81
1 46 52 27 51
1 46 0 31 81
1 21 70 13 18
1 19 97 43 1
1 25 64 21 35
1 28 110 8 128
1 34 34 43 106
1 40 6 45 121
1 8 43 3 37
1 39 16 1 69
1 11 61 37 96
1 31 51 43 85
1 8 77 39 111
1 10 89 42 53
1 15 85 30 37
1 25 117 22 22
1 9 90 3 60
1 38 56 5 104
1 34 119 8 0
1 9 86 43 51
1 21 35 29 36
1 43 30 50 124
1 14 9 4 39
1 32 89 37 45
1 22 34 49 48
1 34 31 43 22
1 2 22 9 102
1 8 115 39 128
1 14 48 25 58
1 11 31 23 121
1 32 38 38 47
1 32 43 36 85
1 34 98 41 120
1 46 130 31 72
1 12 129 10 21
1 4 108 5 52
1 33 95 8 19
1 10 123 9 27
1 45 108 26 106
1 33 113 40 6
1 10 43 30 83
1 39 116 7 111
1 14 84 31 110
1 1 36 4 68
1 43 70 9 125
1 31 108 28 55
1 48 113 50 12
1 2 13 46 2
1 23 116 42 21
1 4 108 35 90
1 7 0 39 25
1 26 100 38 101
1 21 23 30 40
1 12 122 9 53
1 4 51 34 130
1 23 69 2 122
1 22 8 9 79
1 30 42 27 45
1 4 129 46 20
1 29 69 31 59
1 23 130 6 6
1 15 25 1 49
1 41 16 22 7
1 42 58 33 17
1 12 36 4 32
1 28 103 39 37
1 43 103 14 63
1 2 86 5 106
1 50 43 23 15
1 30 112 36 116
1 22 129 32 73
1 15 46 17 15
1 27 116 36 31
1 8 53 17 76
1 25 100 14 0
1 41 112 7 87
1 44 130 33 12
1 19 108 22 31
1 27 110 19 105
1 27 94 13 63
1 18 9 5 110
1 29 121 21 44
1 50 28 13 48
1 31 78 18 125
1 32 91 45 2
1 32 89 10 121
1 25 60 38 45
1 8 89 24 102
1 23 80 15 26
1 29 94 45 36
1 10 45 48 22
1 46 19 28 130
1 4 6 23 85
1 28 63 20 83
1 42 129 45 21
1 47 75 6 5
1 33 37 41 10
1 31 91 35 27
1 17 86 8 83
1 4 67 49 35
1 36 122 34 93
1 35 83 19 100
1 45 94 40 96
1 11 101 45 56
1 10 130 30 109
1 29 97 27 71
1 5 68 16 119
1 42 111 34 10
1 30 15 5 130
1 49 124 2 59
1 27 78 19 76
1 30 70 17 65
1 28 28 35 63
1 44 12 35 34
1 37 72 48 102
1 40 74 5 90
1 50 78 38 7
1 48 27 21 116
1 44 102 19 60
1 46 109 32 0
1 35 20 17 103
1 44 51 15 26
1 33 97 8 128
1 2 38 36 66
1 22 69 16 55
1 12 55 5 113
1 49 129 25 63
1 13 104 31 17
1 45 53 5 26
1 22 21 7 66
1 33 111 24 8
1 5 36 6 73
1 41 100 50 123
1 31 32 26 114
1 40 20 43 125
1 16 121 36 75
1 18 107 32 51
1 31 93 12 75
1 37 40 12 129
1 40 30 8 44
1 19 126 3 56
1 36 53 41 20
1 2 88 27 17
1 17 108 28 54
1 50 27 21 129
1 9 126 38 9
1 39 9 38 83
1 50 32 29 4
1 14 29 20 100
1 23 1 15 112
1 22 44 42 95
1 4 47 39 9
1 2 65 7 84
1 42 19 9 27
1 47 115 39 108
1 15 34 1 39
1 24 60 47 82
1 1 107 18 48
1 24 61 47 58
1 29 91 30 103
1 45 84 39 72
1 29 123 35 21
1 34 47 13 6
1 31 122 45 124
1 23 16 3 6
1 48 40 8 42
1 31 101 30 116
1 29 79 25 37
1 24 58 35 128
1 43 64 9 120
1 7 78 27 126
1 46 48 38 38
1 49 116 25 34
1 27 22 35 61
1 45 66 22 7
1 34 107 21 68
1 1 10 12 120
1 35 15 27 68
1 34 39 48 118
1 26 95 4 90
1 50 14 23 40
1 16 25 21 18
1 44 62 31 121
1 28 21 39 37
1 33 57 42 32
1 35 102 21 42
1 23 94 10 47
1 15 2 29 102
1 12 118 37 10
1 40 19 2 61
1 44 90 34 45
1 46 53 42 77
1 12 11 25 77
1 7 21 31 46
1 34 25 28 64
1 17 118 46 128
1 25 35 50 110
1 13 69 3 96
1 4 14 47 100
1 45 120 23 121
1 27 122 13 117
1 3 121 28 127
1 23 78 27 22
1 3 40 50 44
1 11 21 18 27
1 20 15 22 124
1 47 95 14 91
1 1 111 10 45
1 1 23 9 66
1 48 1 15 87
1 4 99 11 76
1 24 83 7 105
1 9 55 40 33
1 24 112 35 24
1 23 80 35 18
1 49 37 10 112
1 2 18 30 84
1 9 53 25 49
1 31 72 20 77
1 43 100 25 52
1 22 99 26 36
1 17 95 34 16
1 43 70 34 2
1 9 11 12 108
1 16 12 43 71
1 20 8 40 109
1 34 20 44 55
1 14 114 30 7
1 16 18 36 45
1 3 123 48 73
1 9 24 5 34
1 10 43 11 75
1 48 31 50 74
1 50 108 6 82
1 29 116 5 68
1 12 17 15 129
1 42 38 27 72
1 4 11 37 130
1 20 86 11 129
1 21 57 14 124
1 7 92 2 114
1 16 12 48 6
1 6 89 22 13
1 25 115 49 70
1 23 74 42 94
1 38 47 16 118
1 38 73 22 72
1 15 39 12 33
1 8 111 14 41
1 12 16 46 110
1 50 14 9 89
1 14 29 45 56
1 26 99 4 33
1 15 107 43 104
1 50 49 25 14
1 36 6 1 97
1 44 63 8 73
1 17 37 30 46
1 18 42 26 88
1 14 110 50 75
1 4 126 31 82
1 20 43 16 102
1 42 124 15 116
1 24 71 43 80
1 19 43 30 123
1 32 106 5 45
1 16 51 33 69
1 9 9 50 126
1 24 79 19 39
1 11 3 50 107
1 19 82 2 56
1 35 36 47 123
1 34 51 17 93
1 26 58 28 20
1 7 50 33 10
1 21 8 44 73
1 2 73 30 43
1 40 59 41 110
1 47 122 34 79
1 50 0 41 31
1 19 37 34 22
1 35 9 12 105
1 30 87 29 74
1 44 81 20 11
1 39 41 38 30
1 16 40 1 38
1 26 36 6 123
1 46 60 35 80
1 20 42 50 21
1 33 15 27 97
1 14 81 28 89
1 42 95 23 84
1 27 67 20 62
1 23 18 32 81
1 45 26 3 31
1 14 47 19 70
1 13 28 11 101
1 17 42 4 4
1 22 63 2 116
1 36 85 17 99
1 22 39 38 114
1 16 45 32 114
1 41 30 50 81
1 2 18 32 123
1 20 98 25 0
1 22 57 15 30
1 47 76 45 130
1 1 108 33 25
1 3 34 38 127
1 38 19 28 64
1 5 79 21 83
1 10 90 32 69
1 33 115 31 121
1 40 41 22 95
1 31 18 35 111
1 43 31 26 91
1 45 4 28 113
1 13 32 22 0
1 10 91 20 127
1 42 65 45 81
1 38 8 9 91
1 28 23 1 96
1 32 120 43 125
1 1 82 4 83
1 32 117 43 17
1 17 32 22 101
1 38 83 4 113
1 45 6 11 81
1 22 4 45 94
1 5 44 29 75
1 5 125 50 73
1 42 37 46 10
1 19 125 33 21
1 33 37 2 125
1 43 82 1 102
1 40 129 29 13
1 8 71 20 88
