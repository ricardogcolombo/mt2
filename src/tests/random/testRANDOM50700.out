0.651123
0.524801
0.430662
0.545444
0.554841
0.526479
0.462305
0.500110
0.578922
0.415027
0.471093
0.579301
0.442455
0.316979
0.583727
0.489974
0.609387
0.427898
0.428228
0.432626
0.300252
0.465335
0.485092
0.532650
0.479721
0.757372
0.635823
0.507605
0.548581
0.523851
0.474726
0.453061
0.438033
0.314168
0.416215
0.554176
0.555600
0.487393
0.569097
0.452086
0.456778
0.539278
0.471019
0.565392
0.460715
0.474418
0.621410
0.368190
0.587928
0.532652
